src/main/java/mybatis/com/mybatis/MapStruct/StudentMapper.java

```diff
@@ -25,30 +26,35 @@ default String convertToFirstName(StudentCreationDto studentCreationDto){
         return studentCreationDto.getFullName().substring(0, studentCreationDto.getFullName().indexOf(" "));
     }
 
-//    default String convertToLastName(StudentCreationDto studentCreationDto){
-//        if (studentCreationDto.getFullName() == null){
-//            return null;
-//        }
-//        return studentCreationDto.getFullName().substring(studentCreationDto.getFullName().indexOf(" ") + 1);
-//    }
-
-//    @Mapping(source = "id", target = "id")
-//    @Mapping(target = "fullName", expression = "java(convertToFullName(studentEntity.getFirstName(), studentEntity.getLastName()))")
-//    @Mapping(source = "gender", target = "gender")
-//    @Mapping(source = "age", target = "age")
-//    @Mapping(source = "subjectEntityList", target = "subjectEntityList")
-//    StudentDto toDto(StudentEntity studentEntity);
-
-//    @Named("toFullName")
-//    default String convertToFullName(String firstName, String lastname){
-//        String fullName = firstName;
-//        fullName += " ";
-//        fullName += lastname;
-//        return  fullName;
-//    }
-
-//    List<StudentDto> toDtoList(List<StudentEntity> studentEntityList);
-
+    default String convertToLastName(StudentCreationDto studentCreationDto){
+        if (studentCreationDto.getFullName() == null){
+            return null;
+        }
+        return studentCreationDto.getFullName().substring(studentCreationDto.getFullName().indexOf(" ") + 1);
+    }
 
+    @Mapping(source = "id", target = "id")
+    @Mapping(target = "fullName", expression = "java(convertToFullName(studentEntity.getFirstName(), studentEntity.getLastName()))")
+    @Mapping(source = "gender", target = "gender")
+    @Mapping(source = "age", target = "age")
+    @Mapping(source = "subjectEntityList", target = "subjectDtoList")
```

Comment:
In previous PR we removed List Suffix from entity and instead started using plurals like entities/subjects. 
Why is the change not reflecting over here? Is the DTO PR tested properly? If not, please test it. 
Always raise PR only after testing all the changes

Reply:
done